rhel8/profiles/ospp.profile

```diff
@@ -106,94 +304,532 @@ selections:
+    ## FTA_TAB.1 Default TOE access banners
+
+    ### FTA_TAB.1.1: https://www.niap-ccevs.org/MMO/PP/-424-/#FTA_TAB.1.1
+    ### Before establishing a user session, the OS shall display an
+    ### advisory warning message regarding unauthorized use of the OS.
+
+
+    #######################################################
+    # Appendix B Selection-Based Requirements
+
+    #######################################################
+    ## FCS_DTLS_EXT.1 DTLS Implementation
+
+    ### FCS_DTLS_EXT.1.1: https://www.niap-ccevs.org/MMO/PP/-424-/#FCS_DTLS_EXT.1.1
+    ### The OS shall implement the DTLS protocol in accordance with
+    ### [selection: DTLS 1.0 (RFC 4347), DTLS 1.2 (RFC 6347)].
+
+
+    ### FCS_DTLS_EXT.1.2: https://www.niap-ccevs.org/MMO/PP/-424-/#FCS_DTLS_EXT.1.2
+    ### The OS shall implement the requirements in TLS (FCS_TLSC_EXT.1) for the DTLS
+    ### implementation, except where variations are allowed according to DTLS
+    ### 1.2 (RFC 6347).
+
+    #######################################################
+    ## FCS_TLSC_EXT.2 TLS Client Protocol
+
+    ### FCS_TLSC_EXT.2.1: https://www.niap-ccevs.org/MMO/PP/-424-/#FCS_TLSC_EXT.2.1
+    ### The OS shall present the Supported Elliptical Curves Extension in the Client
+    ### Hello with the following NIST curves: [selection: secp256r1, secp384r1,
+    ### secp521r1].
+
+
+    #######################################################
+    # Appendix C Objective Requirements
+
+    #######################################################
+    ## FCS_TLSC_EXT.3 TLS Client Protocol
+
+    ### FCS_TLSC_EXT.3.1: https://www.niap-ccevs.org/MMO/PP/-424-/#FCS_TLSC_EXT.3.1
+    ### The OS shall present the signature_algorithms extension in the Client Hello
+    ### with the supported_signature_algorithms value containing the following hash
+    ### algorithms: [selection: SHA256, SHA384, SHA512] and no other hash algorithms.
+
+
+    #######################################################
+    ## FPT_SRP_EXT.1 Software Restriction Policies
+
+    ### FPT_SRP_EXT.1.1: https://www.niap-ccevs.org/MMO/PP/-424-/#FPT_SRP_EXT.1.1
+    ### The OS shall restrict execution to only programs which match an
+    ### administrator-specified [selection:
+    ###     - file path,
+    ###     - file digital signature,
+    ###     - version,
+    ###     - hash,
+    ###     - [assignment: other characteristics]]
+
+
+    #######################################################
+    ## FPT_W^X_EXT.1 Write XOR Execute Memory Pages
+
+    ### FPT_W^X_EXT.1.1: https://www.niap-ccevs.org/MMO/PP/-424-/#FPT_W^X_EXT.1.1
+    ### The OS shall prevent allocation of any memory region with both write
+    ### and execute permissions except for [assignment: list of exceptions]
+
+
+
+
+
+
+
+
+
+    #######################################################
+    ## Rules that need a home?
+
+
+#    - sysctl_kernel_yama_ptrace_scope
+#    - sysctl_kernel_kptr_restrict
```

Comment:
Will we keep the rules that don't have a home in the profile? If there isn't appropriate category for them in OSPP, should they be removed?

Reply:
Don't know. For now, commented out rules that I didn't know what to do with.... IIRC there are some "catch all" categories we can map rules into at our discretion.